pkg/controller/controller.go

```diff
@@ -112,7 +112,7 @@ func GetRESTHandlers(ctx *context.Provider, opts ...Opt) ([]rest.Handler, error)
 	}
 
 	// route REST operation
-	routeOp, err := route.New(ctx)
+	routeOp, err := route.New(ctx, restAPIOpts.autoAccept)
```

Comment:
I'm wondering whether "autoAccept" should be based on features (did-exchange, route, etc), rather than one for all. 

@troyronda @sudeshrshetty